src/Gui/Splashscreen.cpp

```diff
@@ -790,6 +790,8 @@ void AboutDialog::copyToClipboard()
 
     QClipboard* cb = QApplication::clipboard();
     cb->setText(data);
+    
+    ui->copyButton->setText(QString::fromStdString("copied"));

```

Comment:
`About info copied to clipboard` perhaps ?

Reply:
I mean... if you insist. But it is really hard to misinterpret what's going on here:

![image](https://github.com/user-attachments/assets/f981d6af-8eb3-43c7-b110-d1494af52a5f)

![image](https://github.com/user-attachments/assets/64115133-6390-44ba-a4ed-dbcd1ca9c826)